core/common/src/main/AndroidManifest.xml

```diff
@@ -14,7 +14,6 @@
      See the License for the specific language governing permissions and
      limitations under the License.
 -->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.google.samples.apps.nowinandroid.core.common">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
 </manifest>
```

Comment:
Nit: the unused android namespace can be omitted and the `manifest` tag can be collapsed.
```xml
<manifest />
```